### PR TITLE
Feature/filestorage securityservices list

### PR DESCRIPTION
**Pull request by ehdou**
Add support for Security Service List
For #131

### CODE REVIEW

**Comment by ehdou** [comment]
[List function](https://github.com/openstack/manila/blob/16c522bf5e5c3d37018caef11dd7852462193327/manila/api/v1/security_service.py#L89) for security services

**Comment by ehdou** [comment]
@jrperritt This PR is ready for review. It's based on [PR#133](https://github.com/gophercloud/gophercloud/pull/133) and adds support for security service List

**Comment by ehdou** [comment]
@jrperritt this PR has been rebased and is ready for review

File: openstack/sharedfilesystems/v2/securityservices/requests.go [hunk 1]
```diff
@@ -65,3 +68,54 @@ func Delete(client *gophercloud.ServiceClient, id string) (r DeleteResult) {
...
+// securityservices.List function.
+type ListOpts struct {
+	// admin-only option. Set it to true to see all tenant security services.
+	AllTenants bool `q:"all_tenants"`
+	// The security service ID
+	ID string `q:"id"`
+	// The security service domain
+	Domain string `q:"domain"`
+	// The security service type. A valid value is ldap, kerberos, or active_directory
+	Type string `q:"type"`
```
Comment (jrperritt) [inline openstack/sharedfilesystems/v2/securityservices/requests.go]: s/`string`/`SecurityServiceType` ?

Comment (ehdou) [inline openstack/sharedfilesystems/v2/securityservices/requests.go]: Fixed

**Comment by ehdou** [comment]
@jrperritt I fixed your comment, the PR can be reviewed again

**Comment by jrperritt** [comment]
This looks good

**Pull request merged**